HACKING

```diff
@@ -658,7 +658,7 @@ https://github.com/universal-ctags/ctags
 
 Method
 ``````
-* Add foo.c to SRCS in Makefile.am and meson.build.
+* Add foo.c to SRCS in ctags/Makefile.am and meson.build.
```

Comment:
It's in `parsers` inside `ctags/Makefile.am`
```suggestion
* Add ctags/parsers/foo.c to ``parsers`` in ctags/Makefile.am and to the
  ``ctags`` library in meson.build.
```

Reply:
Done, thanks.